contracts/src/avs/OmniAVS.sol

```diff
@@ -138,6 +139,7 @@ contract OmniAVS is IOmniAVS, IOmniAVSAdmin, OwnableUpgradeable, PausableUpgrade
         Operator[] memory ops = _getOperators();
         omni.xcall{ value: msg.value }(
             omniChainId,
+            ConfLevel.Finalized,
```

Comment:
Note AVS contract changes here, which differs from version live on Mainnet. We will be upgrading the AVS regardless